client/src/pages/Login/Login.tsx

```diff
@@ -39,7 +39,7 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid className={classes.paper} item component={Paper} square>
```

Comment:
Just as above, consider the usage of a MUI Paper component and maybe give it an elevation? :)

Ref: https://material-ui.com/components/paper/